src/EditorFeatures/TestUtilities2/Intellisense/ModernCompletionTestState.vb

```diff
@@ -37,6 +39,8 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
                 includeFormatCommandHandler,
                 workspaceKind:=workspaceKind)
 
+            TextView.Options.GlobalOptions.SetOptionValue(Of Integer)(ResponsiveCompletionThresholdOption, 20000)
```

Comment:
Add a comment saying we're increasing the threshold instead of just disabling the feature so that way we're still testing the "real" codepaths. I think it's a good argument so I don't want you to lose it.

Reply:
updated the comment. thank you!